base_no_translation/README.rst

```diff
@@ -0,0 +1,67 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=========================
+Disable model translation
+=========================
+
+Provides means to globally disable translation of field contents for
+all fields on a per Model base.
+Overrides 'translate=True' of all fields of a model (including inherited
+custom fields of other models).
+
+For example to disable translation of all product data (description etc.).
+Inherit from 'product.product' (and 'product.template') and set class
+attribute 'self._disable_field_translations' of the model to False.
+
+
+Installation
+============
+
+No specific requirements.
+
+
+Configuration
+=============
+
+To disable translation of all fields for a given model you need to inherit
+that model and set class attribute 'self._disable_field_translations'
+of the model to False.
```

Comment:
This sounds very counter intuitive: there is a double negative here, I'd expect having to set `disable_field_translations` to `True` in order to disable the translations.